tests/RespTheme/mobilemainrowBackgroundCest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+use \page\RespTheme\Customizer;
+use \page\RespTheme\LogInAndLogOut;
+
+
+class mobilemainrowBackgroundCest
+{
+    public function _before(RespThemeTester $I)
+    {
+    }
+
+    // tests
+    public function tryToTest(RespThemeTester $I,LogInAndLogOut $loginAndLogout, Customizer $customizer)
+    {
+        $I->amGoingTo('Login as Admin');
+        $loginAndLogout->userLogin($I);
```

Comment:
use amGoingTo function before starting automation to show what exactly you are doing